oxytcmri/domain/use_cases/compute_lesions_volumes.py

```
@@ -0,0 +1,179 @@
+from __future__ import annotations
+
+from abc import ABC
+from dataclasses import dataclass
+from typing import Optional, List, cast
+
+from oxytcmri.domain.entities.mri import DTIMetric, MRIExamId, MRIExam, Atlas, RegionOfInterest, AbnormalValueType, \
+    DTIAbnormalValues, AbnormalVoxelData, Mask
+from oxytcmri.domain.entities.subject import Subject
+from oxytcmri.domain.ports.monitoring import EventDispatcher
+from oxytcmri.domain.ports.repositories import RepositoriesRegistry, SubjectRepository, MRIExamRepository, \
+    AtlasRepository, CenterRepository, RegionOfInterestRepository, Repository
```

Ruff (0.11.9)

`oxytcmri.domain.ports.repositories.CenterRepository` imported but unused

Remove unused import: `oxytcmri.domain.ports.repositories.CenterRepository`

(F401)